### ISSUE TITLE
Make SEND_TIMEOUT in PostActivitySaga configurable

### ISSUE THREAD

**Issue by shmuelgutman** [Microsoft/BotFramework-WebChat#1194]
**Branch: Preview**

In some use case my bot send multiple messages in a row and with typeing-delay of 2 seconds, the entire turn takes more than 5 seconds.

Can we make this configurable?
https://github.com/Microsoft/BotFramework-WebChat/blob/e6eed9b80c8c3b942a651a75f8d8c9f0931777b6/packages/core/src/sagas/postActivitySaga.js#L27

**Comment by compulim** [comment]
Will fix in #1260.

## Usage

### Via React

```jsx
<Composer sendTimeout={ 20000 } />
```

### Via Redux store

```jsx
store.dispatch({ type: 'WEB_CHAT/SET_SEND_TIMEOUT', payload: { sendTimeout: 20000 } });
```

**Comment by compulim** [comment]
Fixed in #1260.

**Comment by shmuelgutman** [comment]
many thanks @compulim
